src/utils.ts

```diff
@@ -862,34 +862,34 @@ export function makeInterruptibleAsyncInterval(
 
   function wake() {
     const currentTime = clock();
-    const nextScheduledCallTime = lastCallTime + interval;
-    const timeUntilNextCall = nextScheduledCallTime - currentTime;
+    const timeSinceLastWake = currentTime - lastWakeTime;
```

Comment:
If I am reading the code correctly, the very first time this function is called, lastWakeTime is going to be `undefined`, so this timeSinceLastWake is going to be `NaN`. It may be best to default it to 0 in L854.

Reply:
Updated to default to zero.